Sources/Datadog/RUM/RUMMonitor/Scopes/RUMApplicationScope.swift

```diff
@@ -36,6 +37,9 @@ internal class RUMApplicationScope: RUMScope, RUMContextProvider {
     /// RUM Sessions sampling rate.
     internal let samplingRate: Float
 
+    /// The start time of the application, measured in device date. It equals the time of SDK init.
+    private let applicationStartTime: Date
```

Comment:
I think the name can be confused with application launch time. Can we rename it to something like `scopeStartTime` or `sdkInitTime`?

Reply:
Yes, makes sense - for that reason I called it `sdkInitTime` in upstream, but in `RUMMonitor` this information was translated to `applicationStartTime`. What proposed could be simpler 👍 - applied in https://github.com/DataDog/dd-sdk-ios/pull/697/commits/99ffde23d08c9bb0aa681e94fb12e2ce18ff29cf to avoid rebasing.